### PR TITLE
APPIOS-2737: Clean up sdk

### PR DESCRIPTION
**Pull request by Pattio**

### CODE REVIEW

File: Sources/PayseraAccountsSDK/AccountsApiRequestRouter.swift [hunk 1]
```diff
@@ -73,7 +73,7 @@ public enum AccountsApiRequestRouter: URLRequestConvertible {
     case deleteUserFromAuthorization(authorizationId: String, userId: String)
     
     // MARK: - Declarations
-    static var baseURLString = "https://accounts.paysera.com/public"
+    static let baseURL = URL(string: "https://accounts.paysera.com/public")!
```
Comment (softwaresaiyajin) [inline Sources/PayseraAccountsSDK/AccountsApiRequestRouter.swift:76]: private

File: Sources/PayseraAccountsSDK/AccountsApiRequestRouter.swift [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import Foundation
 import Alamofire
+import Foundation
 import PayseraCommonSDK
 
 public enum AccountsApiRequestRouter: URLRequestConvertible {
```
Comment (softwaresaiyajin) [inline Sources/PayseraAccountsSDK/AccountsApiRequestRouter.swift:5]: Should this (and other routers from other sdks) be declared as public? These cases are being mirrored by AccountsApiClient, anyway. What are your thoughts ?

Comment (Pattio) [reply Sources/PayseraAccountsSDK/AccountsApiRequestRouter.swift:5]: Good point, I'll make them internal

File: Sources/PayseraAccountsSDK/AccountsApiRequestRouter.swift (hunk 1 above)
Comment (Pattio) [reply Sources/PayseraAccountsSDK/AccountsApiRequestRouter.swift]: Done

File: Sources/PayseraAccountsSDK/AccountsApiRequestRouter.swift (hunk 2 above)
Comment (Pattio) [reply Sources/PayseraAccountsSDK/AccountsApiRequestRouter.swift]: Done

**Review by softwaresaiyajin — APPROVED** [review]

**Review by mildamauricaite — APPROVED** [review]

**Pull request merged**